### PR TITLE
tests: unit tests for info and close commands

### LINKED ISSUE #70: cli: create tests for existing functionalities and commands
After #3 is done, we should test all the functionalities and commands we already have. The following list should summarize the different components we need to test:

- [x] Utils functions
- [x] Validation functions
- [x] `root` functions and general way to test command output
- [x] `close` command
- [x] `du` command
- [x] `info` command
- [ ] `list` command
- [x] `logs` command
- [x] `open` command
- [ ] `status` command
- [x] `ls` command

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #70

### CODE REVIEW

File: cmd/root_test.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 func testCmdRun(
 	t *testing.T,
 	cmd, serverPath, serverResponse string,
-	wantError bool, expectedMsgs []string,
+	statusCode int, expectedMsgs []string,
 	args ...string,
```
[the remark this answers is not in the document]
Comment (BrunoRosendo) [reply cmd/root_test.go:28]: You mean replacing the struct definitions by a predefined struct used for every test? Sounds good

[the remark this answers is not in the document]
Comment (BrunoRosendo) [reply cmd/root_test.go:28]: I'm also thinking about creating a struct for the `testCmdRun` params, the number of parameters is increasing and some of them don't need to be used by every test

[the remark this answers is not in the document]
Comment (mdonadoni) [reply cmd/root_test.go:28]: > You mean replacing the struct definitions by a predefined struct used for every test? Sounds good

Yes, exactly!

> I'm also thinking about creating a struct for the testCmdRun params

I think we can use a single struct for both passing the parameters and as a predefined struct for defining tests, what do you think?

[the remark this answers is not in the document]
Comment (BrunoRosendo) [reply cmd/root_test.go:28]: Yes, that's perfect

**Review by mdonadoni — APPROVED** [review]
Nice!

**Comment by mdonadoni** [comment]
Just one last thing, could you please reduce the line length of the title of the last commit? You can remove `used in all commands`, keeping only `tests: replace testCmdRun params with TestCmdParams struct`.

**Pull request merged**